src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsMigrationServiceTest.kt

```diff
@@ -1015,7 +1015,7 @@ fun aNomisIncidentResponse() =
   )
 
 fun aDPSIncidentMigrateResponse() =
-  IncidentMigrateResponse(4321)
+  Incident("4321")
```

Comment:
Will probably expect a String returned (possibly uuid - but not yet confirmed)

Reply:
regardless you may as well treat it as a string even f it is a long